src/devices/cpu/t11/t11.cpp

```diff
@@ -218,27 +227,35 @@ static const struct irq_table_entry irq_table[] =
 // PSW11 also masks IRQ1.  PSW10 also masks ACLO.
 void k1801vm1_device::t11_check_irqs()
 {
+	int8_t              mcir = MCIR_NONE;
+	uint16_t            vsel = 0;
+
```

Comment:
You seem to have removed mcir and vsel from the state, but you didn't actually remove them from the class?  Or am I misreading something